src/store/modules/product/actions.ts

```diff
@@ -84,6 +84,35 @@ const actions: ActionTree<ProductState, RootState> = {
     commit(types.PRODUCT_ADD_TO_UPLD_PRDTS, { product: payload })
   },
 
+  async updateVarianceCount({ commit }, payload) {
```

Comment:
If we are not doing anything related to state, we should make it an action

Reply:
Made it a method